src/components/Instructions.component.js

```diff
@@ -1,18 +1,36 @@
 import React, { Component } from 'react'
-// TODO: uncomment this for Part 1
-// import PropTypes from 'prop-types'
+import PropTypes from 'prop-types'
 import './../styles/instructions.css'
 
 class Instructions extends Component {
+  static propTypes = {
+    shouldDisplayImage: PropTypes.bool,
+    items: PropTypes.arrayOf(PropTypes.string),
+  }
   render() {
     return (
       <div className="instructions">
         Follow the instructions on the README to get started!
-        <img
+        {this.props.shouldDisplayImage ? (
+          <img
           className="instructions__logo"
           src="https://uiuc.hack4impact.org/img/colored-logo.png"
           alt="h4i logo"
-        />
+          />
+        ) : (
+          <div />
+        )}
+        <ul>
+          {this.props.items.filter(function(item){
+            return item.length >= 3
```

Comment:
you can use the arrow function syntax to one line this:

the anonymous function can be rewritten in this syntax:
```javascript
(item) => {
  return item.length >= 3
}
```

arrow functions have an implicit `return` if don't use `{ }` to evaluate an entire block of code. since your `return` statement inside the callback is very short, you should make use of the implicit `return`:

```javascript
(item) => item.length >= 3
```

minor detail, but if you only have one argument, you can ditch the parentheses.

all together, you get something a lot more concise, clear, and expressive:
```javascript
this.props.items.filter(item => item.length >= 3)
```